tiatoolbox/wsicore/wsimeta.py

```diff
@@ -103,7 +103,8 @@ def __init__(
         mpp: Optional[Sequence[float]] = None,
         file_path: Optional[Path] = None,
         raw: Optional[Mapping[str, str]] = None,
-    ):
+    ) -> None:
+        """Initializas WSIMeta."""
```

Comment:
```suggestion
        """Initialize WSIMeta."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/c2e1cd4ccfd6aaeb32664f23d4f6634cddb90090